e2e/mock-api/project.json

```diff
@@ -10,6 +10,7 @@
       "options": {
         "platform": "node",
         "outputPath": "dist/e2e/mock-api",
+        "outputFileName": "index.cjs",
```

Comment:
this is required so that the output of the serve looks for this file when it runs with esbuild